stylesheets/utils/utilities.css

```diff
@@ -33,4 +33,12 @@
 
 .squared {
     border-radius: var(--br-base);
+}
+
+.navigation-iframe {
+    width: 1000px; 
```

Comment:
i think width shouldn't be fixed to 1000px, this can change depending on screen size